docs/reference/src/language/builtins/elements.md

```diff
@@ -656,6 +656,78 @@ export component Example inherits Window {
 }
 ```
 
+## `SwipeGestureRecognizer`
+
+The `SwipeGestureRecognizer` is used to recognize swipes gesture on the screen in some particular direction.
+Press events on it are still forwarded to the children, but with a small delay.
+If the cursor has moved by more than some internal threshold, a swipe gesture is recognized, and events are no
+longer forwared to the children.
```

Comment:
A couple of edits here, more crucially:

- Instead of saying that gestures on the screen are recognised, emphasise that recognition is limited to the recognizer's area.
- This is is typically used for touch input, the use of the word "cursor" may be confusing.
- An internal threshold is not very useful information to provide. I've copied the 8 px we document for the flickable, since that's the same.

```suggestion
Use the `SwipeGestureRecognizer` to react to swipes gesture in some particular direction. Recognition is limited to the element's geometry.

Specify the different swipe directions you'd like to recognise by setting the `swipe-left/right/up/down` properties and react to the gesture in the `swiped` callback.

Pointer press events on the recognizer's area are forwarded to the children with a small delay.
If the pointer moves by more than 8 logical pixels in one of the enabled swipe directions, the gesture is recognized, and events are no longer forwarded to the children.
```

Reply:
> An internal threshold is not very useful information to provide. I've copied the 8 px we document for the flickable, since that's the same.

On the other hand, this documents and therefore fixes it in stone for further versions.

Reply:
It's already documented for Flickable. I think the "risk" of breakage after changing the threshold is the same, regardless of whether it's documented or not. It's a user-facing thing after all.